### PR TITLE
FAT-1239

### PR DESCRIPTION
**Pull request by nayimovag**
[C17001](https://foliotest.testrail.io/index.php?/cases/view/17001)

https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/848/

### CODE REVIEW

**Review by IhorBohdan — APPROVED** [review]

**Review by OllIsanbaev — APPROVED** [review]

File: cypress/e2e/circulation-log/marked-as-missing.cy.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          .then((res) => {
+            servicePointId = res[0].id;
+          });
+        cy.getUsers({
+          limit: 1,
+          query: `"personal.lastName"="${user.username}" and "active"="true"`
+        })
+          .then(() => {
+            UserEdit.addServicePointViaApi(servicePointId, user.userId);
+            cy.getUserServicePoints(user.userId);
```
Comment (TetianaParanich) [inline cypress/e2e/circulation-log/marked-as-missing.cy.js:44]: I think you don't use response from this function in your test

File: cypress/e2e/circulation-log/marked-as-missing.cy.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        user = { ...userProperties };
+        cy.login(user.username, user.password, { path: TopMenu.circulationLogPath, waiter: SearchPane.waitLoading });
+        ServicePoints.getViaApi({ limit: 1, query: 'pickupLocation=="true"' })
+          .then((res) => {
+            servicePointId = res[0].id;
+          });
+        cy.getUsers({
+          limit: 1,
+          query: `"personal.lastName"="${user.username}" and "active"="true"`
+        })
```
Comment (TetianaParanich) [inline cypress/e2e/circulation-log/marked-as-missing.cy.js:41]: cy.getUsers is unnecessary function, you you don't get response

File: cypress/e2e/circulation-log/marked-as-missing.cy.js (hunk 1 above)
Comment (nayimovag) [reply cypress/e2e/circulation-log/marked-as-missing.cy.js]: fixed

File: cypress/e2e/circulation-log/marked-as-missing.cy.js (hunk 2 above)
Comment (nayimovag) [reply cypress/e2e/circulation-log/marked-as-missing.cy.js]: fixed

File: cypress/e2e/circulation-log/marked-as-missing.cy.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+describe('circulation log', () => {
+  before('creating user and checking out item', () => {
+    cy.createTempUser([
+      permissions.inventoryAll.gui,
+      permissions.circulationLogAll.gui,
+    ])
+      .then(userProperties => {
+        user = { ...userProperties };
+        cy.login(user.username, user.password, { path: TopMenu.circulationLogPath, waiter: SearchPane.waitLoading });
```
Comment (TetianaParanich) [inline cypress/e2e/circulation-log/marked-as-missing.cy.js:33]: This login is unnecessary because you use only api methods

Comment (nayimovag) [reply cypress/e2e/circulation-log/marked-as-missing.cy.js]: fixed

**Pull request merged**